app.py

```diff
@@ -95,42 +110,45 @@ async def add_user():
                 last_insert_id = cur.lastrowid
                 await cur.close()
                 pool.close()
-                return jsonify({'message': 'User created successfully!', 'last_insert_id': last_insert_id}), 201
+                return jsonify({"message": "User created successfully!", "last_insert_id": last_insert_id}), 201
     return {"error": "Request must be JSON"}, 415
 
 
 # Check if user exists and return user
-async def check_user_exist(id="", name=""):
+async def check_user_exist(id=None, name=None):
     pool = await create_pool()
     async with pool.acquire() as conn:
         async with conn.cursor() as cur:
-            if id == "":
+            if not id:
                 await cur.execute("SELECT * FROM users WHERE user_name = %s", (name,))
             else:
-                await cur.execute("SELECT * FROM users WHERE user_id = %s", (id,))
+                await cur.execute("SELECT user_id, user_name, email FROM users WHERE user_id = %s", (id,))
             user = await cur.fetchone()
             await cur.close()
             pool.close()
             return user
 
 
 # Update User
-@app.put('/users/<int:user_id>')
+@app.put("/users/<int:user_id>")
 async def update_user(user_id):
-    if await check_user_exist(id=user_id) is None:
+    if not await check_user_exist(id=user_id):
         return {"error": "User not found"}, 404
     else:
         if request.is_json:
             update_user = request.get_json()
-            update_user["password"] = generate_password_hash(update_user["password"], method="sha256")
-            if update_user['user_name'] == '' or update_user['email'] == '' or update_user['password'] == '':
-                return {"error": "Feild is empty"}, 400
+            required_fields = ["user_name", "email", "password"]
+            for field in required_fields:
+                if field not in update_user:
```

Comment:
From convention, `for field in required_fields:` is not used by Python Developers. The reason is the variable called `field` is only available inside the `for` block. So just to convince to the other developers not to use it outside the `for` loop block, Python developers tends to make it as protected as bellow.

`for _field in required_fields:`